src/cli.js

```diff
@@ -38,13 +32,27 @@ program
   .option(
     "--ignore <pattern>",
     "adds a glob pattern used to ignore input files (repeatable)",
-    listOption
+    listOption,
+    []
   )
   .option(
     "--files <pattern>",
-    "adds a glob pattern used to find input files (repeatable)",
-    listOption
-  );
+    "glob pattern used to find input files",
+    "**/*.{js,jsx,tsx}"
+  )
+  .option(
+    "--sort <alphabetical|usage>",
+    "specify sort type of the report",
+    /^(alphabetical|usage)$/i,
+    "usage"
+  )
+  .option(
+    "--report <usage|props|children>",
+    "specify reports to show (can be repeted)",
```

Comment:
When you add default values to `.option(...)` it actually shows that in the --help output, so you'll need to remove the `[process.cwd()]` default because it will look wrong in that output. That's why I was putting the default values outside of the option parser, since I didn't actually want any of them to show up in `--help`